documentation/source/directives.rst

```diff
@@ -106,6 +108,7 @@ define. It behaves the same as the doxygenstruct directive.
 
 Checkout the :ref:`example <define-example>` to see it in action.
 
+.. _doxygenenum:
```

Comment:
The header below this is `doxygenconcept`, not enum.

Reply:
Fixed and added `doxygenenum` above the actual directive, where it was missing. (will push soon)